protocol/proto/openyolo.proto

```diff
@@ -23,42 +23,45 @@ option java_outer_classname = "Protobufs";
 
 message CredentialRetrieveRequest {
     // at least one authMethod required
-    repeated string authMethods = 1;
-    map<string, TokenRequestInfo> supportedTokenProviders = 2;
-    map<string, bytes> additionalProps = 3;
+    repeated AuthenticationMethod authMethods = 1;
+
+    map<string, TokenRequestInfo> supportedTokenProviders = 3;
+    map<string, bytes> additionalProps = 4;
 }
 
-message CredentialRetrieveResponse {
+message CredentialRetrieveBbqResponse {
```

Comment:
My rationale for using a different word here is that "results" are the terminal messages, and "response" was intermediary. Pretty weak reasoning, I suppose, but as this is hidden from the API I don't know if it matters all that much.